versioneer.py

```diff
@@ -340,6 +339,7 @@
 """
 
 from __future__ import print_function
+
```

Comment:
Looks like change in `versioneer.py` are unrelated to this PR main topic.
Can you revert those changes?